go/engine/wot_test.go

```diff
@@ -9,65 +9,88 @@ import (
 )
 
 func TestWebOfTrust(t *testing.T) {
-	tc := SetupEngineTest(t, "wot")
-	defer tc.Cleanup()
+	tc1 := SetupEngineTest(t, "wot")
```

Comment:
this test basically wasn't correct at all, but a server bug was making it look like it worked.

all these changes make the test actually test something useful.

Reply:
is there a ticket for the server bug?